app/src/main/java/com/example/playlistmaker/presentation/search/SearchActivity.kt

```diff
@@ -235,25 +258,35 @@ class SearchActivity : AppCompatActivity() {
         trackAdapter.notifyDataSetChanged()
         alertImage.setImageResource(imageRes)
         alertTextView.text = getString(textRes)
-        alertButton.visibility = if (showUpdateButton) View.VISIBLE else View.GONE
-        alertView.visibility = View.VISIBLE
+        alertButton.visibility = if (showUpdateButton) VISIBLE else GONE
```

Comment:
Лучше оборачивать в { } хотя бы то, что внутри if
```suggestion
        alertButton.visibility = if (showUpdateButton) {
            VISIBLE 
        } else GONE
```